ORBIT/manager.py

```diff
@@ -1015,6 +1200,33 @@ def turbine_capex_per_kw(self):
         _capex = self.config.get("turbine_capex", None)
         return _capex
 
+    @property
+    def total_capex(self):
+        """
+        Returns total BOS CAPEX including commissioning and decommissioning.
```

Comment:
This docstring is  a bit misleading if turbine capex is included.  Seems like this is just total project capex.

Reply:
Updated.